specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-05-01-preview/containerregistry_scopemap.json

```diff
@@ -0,0 +1,916 @@
+        }
+      }
+    },
+    "Token": {
+      "description": "An object that represents a token for a container registry.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TokenProperties",
+          "description": "The properties of the token.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "TokenProperties": {
+      "description": "The properties of a token.",
+      "type": "object",
+      "properties": {
+        "creationDate": {
+          "format": "date-time",
+          "description": "The creation date of scope map.",
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "Provisioning state of the resource.",
+          "enum": [
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "scopeMapId": {
+          "description": "The resource ID of the scope map to which the token will be associated with.",
+          "type": "string"
+        },
+        "objectId": {
+          "description": "The user/group/application object ID for which the token has to be created.",
+          "type": "string"
+        },
+        "permissionsStatus": {
+          "description": "The status of the permission propagation to all the users associated with this token.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the token. E.g. Device.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "TokenUpdateParameters": {
+      "description": "The parameters for updating a token.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TokenProperties",
+          "description": "The properties of the token update parameters.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "TokenListResult": {
+      "description": "The result of a request to list tokens for a container registry.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of tokens. Since this list may be incomplete, the nextLink field should be used to request the next list of tokens.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Token"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next list of tokens.",
+          "type": "string"
+        }
+      }
+    },
+    "ProxyResource": {
```

Comment:
FYI, there are common definitions you can reference in: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json

Reply:
Fixed it.

Reply:
@KrisBash  The parent swagger has some description mismatch with that defined in common types. The build breaks for that. I will fix the parent swagger and this change separate to this PR. For now I added the custom proxy resource schema to the definitions.

Reply:
This is a recommendation, not blocking. thanks